CHANGELOG.md

```diff
@@ -7,14 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## \[Unreleased\]
 
+## [0.34.0](https://github.com/Substra/substrafl/releases/tag/0.34.0) - 2023-01-30
+
 ### Added
 
 - Possibility to test on an organization where no training have been performed (#74)
```

Comment:
Nitpick

```suggestion
- Possibility to test on an organization where no training has been performed (#74)
```

I'm not 100% sure but I think "training" here is an uncountable noun.